demos/projects/ST/b-l475e-iot01a/config/core_http_config.h

```diff
@@ -0,0 +1,149 @@
+/*
+ * coreHTTP v2.1.0
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/**
+ * @file core_http_config_defaults.h
+ * @brief The default values for the configuration macros for the HTTP Client
+ * library.
+ *
+ * @note This file SHOULD NOT be modified. If custom values are needed for
+ * any configuration macro, a core_http_config.h file should be provided to
+ * the HTTP Client library to override the default values defined in this file.
+ * To use the custom config file, the HTTP_DO_NOT_USE_CUSTOM_CONFIG preprocessor
+ * macro SHOULD NOT be set.
+ */
+
+#ifndef CORE_HTTP_CONFIG_DEFAULTS_
+#define CORE_HTTP_CONFIG_DEFAULTS_
+
+/* *INDENT-OFF* */
+#ifdef __cplusplus
+    extern "C" {
+#endif
+/* *INDENT-ON* */
+
+/**
+ * @brief Maximum size, in bytes, of headers allowed from the server.
+ *
+ * If the total size in bytes of the headers received from the server exceeds
+ * this configuration, then the status code
+ * #HTTPSecurityAlertResponseHeadersSizeLimitExceeded is returned from
+ * #HTTPClient_Send.
+ *
+ * <b>Possible values:</b> Any positive 32 bit integer. <br>
+ * <b>Default value:</b> `2048`
+ */
+#ifndef HTTP_MAX_RESPONSE_HEADERS_SIZE_BYTES
+    #define HTTP_MAX_RESPONSE_HEADERS_SIZE_BYTES    2048U
+#endif
+
+/**
+ * @brief The HTTP header "User-Agent" value.
+ *
+ * The following header line is automatically written to
+ * #HTTPRequestHeaders_t.pBuffer:
+ * "User-Agent: my-platform-name\r\n"
+ *
+ * <b>Possible values:</b> Any string. <br>
+ * <b>Default value:</b> `my-platform-name`
+ */
+#ifndef HTTP_USER_AGENT_VALUE
+    #define HTTP_USER_AGENT_VALUE    "my-platform-name"
```

Comment:
maybe something like 
```suggestion
    #define HTTP_USER_AGENT_VALUE    "stml475-adu"
```